Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ConceptMergeController.java

```diff
@@ -0,0 +1,53 @@
+package edu.asu.conceptpower.web;
+
+import java.security.Principal;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.ModelMap;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import edu.asu.conceptpower.app.bean.ConceptsMergeBean;
+import edu.asu.conceptpower.app.core.IConceptManager;
+import edu.asu.conceptpower.app.exceptions.DictionaryDoesNotExistException;
+import edu.asu.conceptpower.app.exceptions.DictionaryModifyException;
+import edu.asu.conceptpower.app.exceptions.IndexerRunningException;
+import edu.asu.conceptpower.app.exceptions.LuceneException;
+import edu.asu.conceptpower.app.service.IConceptMergeService;
+import edu.asu.conceptpower.core.ConceptEntry;
+
+@Controller
+public class ConceptMergeController {
+
+    @Autowired
+    private IConceptManager conceptManager;
+
+    @Autowired
+    private IConceptMergeService conceptMergeService;
+
+    @RequestMapping(value = "prepareMergeConcept")
```

Comment:
url should be ```auth/...``` like all the others.

Also, instead of camel case, I'd use a path. E.g. ```auth/concepts/merge``` and ```auth/concepts/merge/prepare``` (if that doesn't overlap with any other controllers).